src/Aspire.Hosting.Azure/Provisioning/AzureProvisionerExtensions.cs

```diff
@@ -26,7 +26,9 @@ public static IDistributedApplicationBuilder AddAzureProvisioning(this IDistribu
 
         // Attempt to read azure configuration from configuration
         builder.Services.AddOptions<AzureProvisionerOptions>()
-            .BindConfiguration("Azure");
+            .BindConfiguration("Azure")
```

Comment:
OK with this for now but might we want this to be configurable @davidfowl?

Reply:
Not in this PR. The provisioner already hardcodes this section.